pages/advanced-algorithms/available-algorithms/bridges.mdx

```diff
@@ -40,7 +40,7 @@ You can execute this algorithm on [graph projections, subgraphs or portions of t
 
 ### `get()`
 
-The procedure indentifies a bridge relationships in the graph which if deleted,
+The procedure identifies a bridge relationships in the graph which if deleted,
```

Comment:
s/a bridge relationships/a bridge relationship
The "number" should agree. The word "a" implies a single relationship.